swift/ios-shoppe-demo/Views/StoreViewController.swift

```diff
@@ -12,6 +12,28 @@ class StoreViewController: UICollectionViewController, UICollectionViewDelegateF
 
     var products: [Product] = []
 
+    // MARK: - NavigationBar Button Properties
+
+    var barCartButton: UIBarButtonItem {
```

Comment:
Hey @HaroldFS .  I'm curious on the declaration of `barCartButton` and  `cartNumberView`.  It looks like `cartNumberView` creates a `UIBarButtonItem()` in its declaration and later assigns the `tintColor` when it is set.  But in `setupNavigationBar` there's also a `cartNumberView = UIBarButtonItem(...)`.  Would it make sense to init `barCartButton` and `cartNumberView` from a single location like `setupNavigationBar`?

Reply:
`UIBarButtonItem` was being initialized in multiple locations and that was redundant. I removed it from the `setupNavigationBar ` method and left it in the `setCartUI` method. Thanks for the feedback!